### GPT Links: 
- Question 8: https://chatgpt.com/share/670487d1-db14-800e-84e4-600aa87c5626

#### 2. Towards the end of the "first pre-lecture" video (above) it is stated that, "It is important to note that outcomes of tests refer to the population parameter, rather than the sample statistic! As such, the result that we get is for the population." In terms of the distinctions between the concepts of $x_i\!$'s, $\bar x$, $\mu$, and $\mu_0$, how would you describe what the sentence above means? Explain this concisely in your own words for a "non-statsitical" audience, defining the technical statistical terminology you use in your answer.<br>

- $x_i\!$  can be defined as individual values a sample (grade of one person)
- $\bar x$ is the average of the values in a sample (average grade in a sample)
- $\mu$ is the true average for the entire population (the average grade of everyone in the entire class, not just a sample)
- $\mu_0$ is the assumed average that we are testing against (if the average grade is 70, then $\mu_0$ is 70)

When a test gives us a result, it’s only telling us whether or not our sample data (like the$x_i\!$, and $\bar x$) says that the true population average is likely different from 𝜇0. Basically, the outcome of the test is about what is happening to the entire population, not just the sample. A sample is like a subset of the entire population. So the results we get from it only affect the sample

#### 4. The second "Pre-lecture" video (above) describes suggest that a smaller p-value makes the null hypothesis look more ridiculous. Explain why this is in your own words in a way that makes the most sense to you, clarifying the meaning of any technical statistical terminology you use in your answer.

- H0: null hypothesis

When you get a small p-value, it means that the chance of seeing your data under the assumption that H₀ is true is very low. This is because if you have a high p-value (usually >0.05), it means that your data is consistent with the null hypothesis as its more random, supporting the null hypothesis. So, you do not reject the null hypothesis because the result looks like what you'd expect if the null hypothesis is true. If you have a low p-value (usually <0.05), it means your data is unlikely under the null hypothesis. In this case, you would consider rejecting the null hypothesis because the observed result seems too unusual to be just random chance.

#### 8. Complete the following assignment. 

#### Fisher's Tea Experiment

This project is based on an adaptation of the famous "Lady Tasting Tea" experiment by Sir Ronald Fisher, where the ability to distinguish whether milk or tea was poured first is tested. In this modern experiment, we asked 80 STA130 students to identify whether tea or milk was poured first in a cup. Of these students, 49 correctly identified the pouring order. The goal of this analysis is to determine whether the performance of these students can be attributed to random guessing or if their ability to distinguish the order is statistically significant.

1.  In the original experiment, Dr. Muriel Bristol claimed she could differentiate between cups of tea based on whether the milk or tea was poured first. Fisher set up a hypothesis test to evaluate whether her ability to correctly identify the pouring order was due to chance or actual ability.

- The modern experiment with STA130 students retains the core of Fisher's problem, but on a larger scale:

- Population: The population in the original experiment was personalized, focusing on Dr. Bristol's claimed ability. In the modern version, the population consists of a broader group—STA130 students—so the ability being tested is generalized.
- Sample Size: The original experiment involved 8 cups, while in this experiment, 80 students each evaluate one cup.
The statistical principle underlying both experiments is the same: testing whether an observed ability to differentiate between two conditions is significantly different from random guessing.

2. Hypothesis:

- Null Hypothesis (H₀): The proportion of students able to correctly identify whether milk or tea was poured first is no better than random guessing, i.e., 𝑝=0.5

- Alternative Hypothesis (H₁): The proportion of students able to correctly identify the pouring order is greater than 0.5, i.e., 𝑝>0.5

To test the hypotheses, we will conduct a one-proportion z-test to determine if the observed proportion of correct answers (p hat = 0.6125) is significantly greater than the hypothesized proportion under random guessing (p0 = 0.5).

The z-test formula is as follows:

z = \frac{\hat{p} - p_0}{\sqrt{\frac{p_0(1 - p_0)}{n}}}

- p hat  = 0.6125 is the sample proportion,
- p0 = 0.5 is the hypothesized proportion,
- n = 80 is the sample size.

In [1]:
import numpy as np
from scipy import stats

# Reproducibility
np.random.seed(42)

# Given data
n = 80  # sample size
x = 49  # number of correct guesses
p_hat = x / n  # sample proportion
p_0 = 0.5  # hypothesized population proportion under H₀

# Standard error calculation
standard_error = np.sqrt(p_0 * (1 - p_0) / n)

# z-test statistic
z = (p_hat - p_0) / standard_error

# p-value (for a one-tailed test)
p_value = 1 - stats.norm.cdf(z)

# Display results
print(f"z-statistic: {z:.4f}")
print(f"p-value: {p_value:.4f}")

z-statistic: 2.0125
p-value: 0.0221


Based on the z-test results:

- If the p-value is less than 0.05, we conclude that the students' ability to identify whether tea or milk was poured first is statistically significantly better than random guessing.
- If the p-value is greater than 0.05, we fail to reject the null hypothesis, suggesting that students' performance is consistent with guessing.
- The final conclusion will provide evidence either supporting or refuting the hypothesis that STA130 students can correctly identify the pouring order better than by chance.


The z-statistic is approximately 2.01, and the p-value is approximately 0.0221.

Since the p-value (0.0221) is less than the typical significance level of 0.05, we reject the null hypothesis (H₀). This suggests that the students' ability to correctly identify whether tea or milk was poured first is statistically significantly better than random guessing. Thus, the conclusion supports the hypothesis that STA130 students can correctly identify the pouring order better than by chance. ​